pandas/tests/series/test_timeseries.py

```diff
@@ -422,6 +422,38 @@ def test_pct_change_shift_over_nas(self):
         expected = Series([np.nan, 0.5, 0.0, 2.5 / 1.5 - 1, 0.2])
         assert_series_equal(chg, expected)
 
+    @pytest.mark.parametrize(
+        "offset_1, offset_2",
+        product(
+            ["D", "W", "M"],
+            [
+                "B",
+                "C",
+                "W",
+                "M",
+                "SM",
+                "BM",
+                "CBM",
+                "MS",
+                "SMS",
+                "BMS",
+                "CBMS",
+                "Q",
+                "BQ",
+                "BA",
+                "A",
+                "AS",
+            ],
+        ),
+    )
```

Comment:
Instead of using product here just parametrize twice, so:

```python
@pytest.mark.parametrize("offset_1", ["D", "W", "M"])
@pytest.mark.parametrize("offset_2", ...
```

Does the same thing but is more idiomatic to pytest

Reply:
Oh, that's a cool feature I didn't know that. I'll fix it like that.